src/loan_form/LoanForm.jsx

```diff
@@ -22,82 +21,67 @@ export default class LoanForm extends React.Component {
   }
   
   @autobind
+  handleAddLoan() {
+    const newLoans = this.state.loans.concat(
+      this.createLoan()
+    )
+
+    this.setState({ loans: newLoans })
+  }
+  
+  @autobind
   handleChange(id, name, value) {
     const newLoans = this.state.loans
 
+    const dashCaseToCamelCase = string => {
```

Comment:
we should really just use the lodash camleCase function. you can pull in just that module with yarn and update the package.json and yarn.lock files.

also, fwiw, a function like this, which we would likely use elsewhere, should be pulled into its own file so then we can write tests against it.